metrics.yaml

```diff
@@ -0,0 +1,29 @@
+metrics:
+    requests_in_flight:
+        type: gauge
+        description: Number of requests in flight.
+        command: scripts/metrics_scripts/requests_in_flight
```

Comment:
Can't we call the collect_metrics.py with the right argument directly from here, without having to create a script for every metric?